### PR TITLE
[DOC] add link to guide

### PR DESCRIPTION
**Pull request by sciunto**
From the mailing list, I thought useful to put a link from the gallery to the user guide.

Please correct the rst syntax if it's not correct (this syntax makes me allergic). It looks good locally.

### CODE REVIEW

File: doc/examples/README.txt [hunk 1]
```diff
@@ -4,4 +4,6 @@ General examples
 -------------------
 
 General-purpose and introductory examples for the scikit.
+A guide_ introduces conventions and basic image manipulations.
```
Comment (emmanuelle) [inline doc/examples/README.txt]: line break between the two sentences?

I would suggest a slight change 

The <user guide>_ (narrative documentation) introduces conventions and basic image manipulations.

**Comment by emmanuelle** [comment]
Thanks @sciunto, cross-referencing between documentation pages is very useful. I suggest a slight rephrasing, but otherwise, looks good to me. :+1:

**Comment by emmanuelle** [comment]
And the link syntax is correct indeed!

**Comment by sciunto** [comment]
ok, I made edits :)

**Comment by emmanuelle** [comment]
Thanks @sciunto, now we need a second :+1: :-)

**Comment by blink1073** [comment]
LGTM, thanks @sciunto!

**Pull request merged**